server/command.go

```diff
@@ -25,13 +28,19 @@ type CommandHandler struct {
 	defaultHandler CommandHandlerFunc
 }
 
+// Available settings
+const (
+	SETTINGS_NOTIFICATIONS = "notifications"
```

Comment:
0/5 Can we please use the Go and not the Mattermost const naming convention in Jira?